README.md

```diff
@@ -6,3 +6,22 @@ on the
 
 See rendered site at:
 [ngs-docs.github.io/2021-august-remote-computing/](https://ngs-docs.github.io/2021-august-remote-computing/)
+
+## To contribute:
+
+- Clone this repo
+- Create a branch and add changes
+- Enter the following commands to re-build the bookdown (`make`) and view the website (`open`). You only have to create the `bookdown` environment once!:
+   ```
+   mamba env create -f environment.yml -n bookdown
+   conda activate bookdown
+   make
+   open docs/index.html
+   ```
+- Commit changes
+  ```
+  git add .
```

Comment:
```suggestion
  git add docs
```

I think it's dangerous to `git add .` :). But `docs/` is completely (re)built by make.